firebase-config/src/main/java/com/google/firebase/remoteconfig/internal/ConfigLogger.java

```diff
@@ -0,0 +1,63 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package com.google.firebase.remoteconfig.internal;
+
+import com.google.android.datatransport.Event;
+import com.google.android.datatransport.Transport;
+import com.google.firebase.remoteconfig.BuildConfig;
+import com.google.firebase.remoteconfig.proto.ClientMetrics.ClientLogEvent;
+import com.google.firebase.remoteconfig.proto.ClientMetrics.ClientLogEvent.EventType;
+import com.google.firebase.remoteconfig.proto.ClientMetrics.FetchEvent;
+
+public class ConfigLogger {
```

Comment:
does this have to be public? Try to reduce visibility as much as possible. Thanks!

Reply:
Looks like I'd have to keep it public unless I move ConfigLogger out of the /remoteconfig/internal folder into the parent /remoteconfig folder since I instantiate a ConfigLogger object in the RemoteConfigComponent class. What do you think about keeping it public vs. moving the code to its parent folder?

Reply:
Sounds good, thats fine then! anything in /internal is hidden (i.e., not visible to developers) anyways